server/models/executor.js

```diff
@@ -0,0 +1,30 @@
+var async = require('async');
+
+module.exports = function(Executor) {
+  Executor.observe('after save', function(ctx, next) {
+    var serviceManager = Executor.app.serviceManager;
+    if (ctx.instance) {
+      if (serviceManager.onExecutorUpdate.length === 2) {
```

Comment:
I don't think it is worth it if the other checks are scheduled to be removed when the major version bumps.